src/vm/rejit.inl

```diff
@@ -26,7 +26,8 @@ inline BOOL ReJitManager::IsReJITEnabled()
 {
     LIMITED_METHOD_CONTRACT;
 
-    return CORProfilerEnableRejit();
+    DWORD dwRejitOnAttachDisabled = CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_CORECLR_DISABLE_REJIT_ON_ATTACH);
+    return CORProfilerEnableRejit() || !dwRejitOnAttachDisabled;
```

Comment:
While looking around at IsReJITEnabled I noticed two other things that would be good to fix. Both predate your changes, but since this will make ReJIT more common the issues are exacerbated:

https://github.com/dotnet/coreclr/blob/master/src/vm/codeman.cpp#L2541
The alignment is only needed if we are using a JumpStamp and it is no longer true that ReJIT => jumpstamps. Instead we should use: 
`pMD->IsVersionableWithJumpStamp()`

https://github.com/dotnet/coreclr/blob/master/src/vm/jitinterface.cpp#L12401
Same thing, we only need the no-ops if we are using a JumpStamp and not all rejitting will use JumpStamp. Instead:
`ftn->IsVersionableWithJumpStamp()`